xpsi/tools/synthesise.pyx

```diff
@@ -61,7 +61,8 @@ def synthesise_exposure(double exposure_time,
     :param tuple component_phases:
         For each component, a C-contiguous :class:`numpy.ndarray` of phases
         in cycles at which the model :obj:`signal` is evaluated on
-        the interval ``[0,1]``.
+        the interval ``[0,1]``. Typically this is more finely spaced than the
+        to be synthesised data.
```

Comment:
Would it be more clear if saying "the resulting synthesised data" instead of "the to be synthesised data"?

Same comment also fir below, where this text is repeated.